test/ducks/apps/components/applicationPage/__snapshots__/index.spec.js.snap

```diff
@@ -147,7 +147,7 @@ Gestionnaire de photos pour Cozy v3 :tada:",
         namePrefix="Cozy"
         parent="/myapps"
       />
-      <withRouter(Wrapper)
+      <Wrapped
```

Comment:
Nit for later: we should give a display name to our HOCs for better snapshots reading

Reply:
Good idea, thanks: https://github.com/cozy/cozy-client/pull/440